plugins/cli-plugins/package.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "@reframe/cli-plugins",
+    "version": "0.0.1-rc.1"
```

Comment:
We are using Lerna with synced version for all packages. So all packages need to have the same version. I believe it's currently `0.0.1-rc.7`

Reply:
Also every package needs to be added to the `workspaces` field of the `package.json` at the root of the repo